### ISSUE TITLE
Actor update equation in DDPG

### ISSUE THREAD

**Issue by moakra** [germain-hug/Deep-RL-Keras#5]
Hi,

When you use the gradient of the critic to update the actor [here](https://github.com/germain-hug/Deep-RL-Keras/blob/master/DDPG/actor.py#L69), why do you put in the third parameter of tf.gradients() "-action_gdts" instead of "action_gdts". From where does the minus sign come ?

I double checked the formula and I still don't see why it is the case in your code.

Thanks!

**Comment by germain-hug** [comment]
Apologies for the late response, the negation here is a simple trick to compute the gradients using the loss function of the actor's network. This is similar to [OpenAI's baselines](https://github.com/openai/baselines/tree/master/baselines/ddpg)

**Comment by makrout** [comment]
Thanks!
